impl/src/test/java/org/ehcache/impl/internal/store/offheap/AssertingLazyOffHeapValueHolder.java

```diff
@@ -0,0 +1,91 @@
+package org.ehcache.impl.internal.store.offheap;
+
+import org.ehcache.spi.serialization.Serializer;
+import org.terracotta.offheapstore.storage.portability.WriteContext;
+
+import java.nio.ByteBuffer;
+import java.util.Collection;
+import java.util.function.Predicate;
+
+import static java.util.Arrays.asList;
+import static java.util.Arrays.stream;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+public class AssertingLazyOffHeapValueHolder<V> extends LazyOffHeapValueHolder<V> {
```

Comment:
This class could use some comments to future maintainers.

Reply:
Added a block comment on the static field explaining what is going on.